CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/WelcomeTopComponent.java

```diff
@@ -0,0 +1,360 @@
+ */
+package au.gov.asd.tac.constellation.views.welcome;
+
+import au.gov.asd.tac.constellation.graph.file.open.RecentFilesWelcomePage;
+import au.gov.asd.tac.constellation.graph.interaction.plugins.io.screenshot.RecentGraphScreenshotUtilities;
+import au.gov.asd.tac.constellation.security.ConstellationSecurityManager;
+import au.gov.asd.tac.constellation.security.proxy.ProxyUtilities;
+import au.gov.asd.tac.constellation.utilities.BrandingUtilities;
+import au.gov.asd.tac.constellation.utilities.font.FontUtilities;
+import java.awt.BorderLayout;
+import java.io.File;
+import java.util.List;
+import javafx.application.Platform;
+import javafx.embed.swing.JFXPanel;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Orientation;
+import javafx.geometry.Pos;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Cursor;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ContentDisplay;
+import javafx.scene.control.Label;
+import javafx.scene.control.SplitPane;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import org.netbeans.api.settings.ConvertAsProperties;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.util.Lookup;
+import org.openide.util.NbBundle.Messages;
+import org.openide.windows.TopComponent;
+
+/**
+ *
+ * WelcomeTopComponent is designed to inform users of news about Constellation.
+ *
+ * @author canis_majoris
+ */
+@ConvertAsProperties(
+        dtd = "-//au.gov.asd.tac.constellation.visual.welcome//Welcome//EN",
+        autostore = false
+)
+@TopComponent.Description(
+        preferredID = "WelcomeTopComponent",
+        iconBase = "org/netbeans/modules/autoupdate/ui/info_icon.png",
+        persistenceType = TopComponent.PERSISTENCE_NEVER
+)
+@TopComponent.Registration(
+        mode = "editor",
+        openAtStartup = false
+)
+@ActionID(
+        category = "Window",
+        id = "au.gov.asd.tac.constellation.views.welcome.WelcomeTopComponent"
+)
+@ActionReferences({
+    @ActionReference(path = "Menu/Help", position = 0)
+})
+@TopComponent.OpenActionRegistration(
+        displayName = "#CTL_WelcomeTopComponentAction",
+        preferredID = "WelcomeTopComponent"
+)
+@Messages({
+    "CTL_WelcomeTopComponentAction=Welcome Page",
+    "CTL_WelcomeTopComponentTopComponent=Welcome Page",
+    "HINT_WelcomeTopComponentTopComponent=Welcome Page"
+})
+public final class WelcomeTopComponent extends TopComponent {
```

Comment:
This should either extend `SwingTopComponent<Component or extension>` or `JavaFxTopComponent<Pane or extension>` in order to use the view framework